src/app/components_ngrx/custom-query-panel/custom-query-panel.component.less

```diff
@@ -25,57 +22,81 @@
       background: @color-pf-black-700;
     }
   }
-  &__list {
-    &-type {
-      outline: none;
-      padding: 10px;
-      padding-left: 45px;
+  .f8-cf {
+    display: block;
+    color: @color-pf-white;
+    background-color: @color-pf-black-900;
+    &__hdr {
+      display: block;
+      padding: 20px 15px;
       border-left: 4px solid;
       border-left-color: transparent;
-      &-link {
-        display: block;
-        font-size: 15px;
-        color: @color-pf-white;
-        line-height: 15px;
-        text-decoration: none;
-        outline: none;
-        &:hover {
-          color: @color-pf-blue;
-          text-decoration: none;
-          background: @color-pf-black-700;
-        }
-        &:visited, &:focus {
-          text-decoration: none;
-          outline: none;
+      border-bottom: 1px solid;
+      border-bottom-color: @color-pf-black;
+      line-height: 15px;
+      cursor: default; // as item is not clickable
+      &.nohover
+        h3 {
+          color: @color-pf-black-500;
+          line-height: 1;
         }
       }
-      &:hover {
-        color: @color-pf-white;
+      &:not(.nohover):hover {
         background: @color-pf-black-700;
       }
-      .fa {
-        padding: 0 5px;
-        font-size: 16px;
+    }
+    &__list {
+      &-type {
+        outline: none;
+        padding: 10px;
+        padding-left: 45px;
+        border-left: 4px solid;
+        border-left-color: transparent;
+        &-link {
+          display: block;
+          font-size: 15px;
+          color: @color-pf-white;
+          line-height: 15px;
+          text-decoration: none;
+          outline: none;
+          &:hover {
+            color: @color-pf-blue;
+            text-decoration: none;
+            background: @color-pf-black-700;
+          }
+          &:visited, &:focus {
+            text-decoration: none;
+            outline: none;
+          }
+        }
         &:hover {
-          color: @color-pf-blue;
+          color: @color-pf-white;
+          background: @color-pf-black-700;
         }
+        .fa {
+          padding: 0 5px;
+          font-size: 16px;
+          &:hover {
+            color: @color-pf-blue;
+          }
+        }
+      }
+      &:focus {
+        outline: none;
       }
     }
-    &:focus {
+    &--selected {
       outline: none;
+      border-left-color: @color-pf-blue !important;
+      background-color: @color-pf-black-800;
     }
-  }
-  &--selected {
-    outline: none;
-    border-left-color: @color-pf-blue !important;
-    background-color: @color-pf-black-800;
-  }
-  .dropdown-kebab-pf {
-    .btn-link {
-      color: @color-pf-white;
-    }
-    .dropdown-menu-right {
-      right: -10px;
+    .dropdown-kebab-pf {
+      .btn-link {
+        color: @color-pf-white;
+      }
+      .dropdown-menu-right {
+        right: -10px;
+      }
     }
   }
 }
```

Comment:
@sanbornsen You did something wrong with this file while resolving the conflict. @SMahil Can you please look into it.

Reply:
sure

Reply:
CSS is fixed in https://github.com/fabric8-ui/fabric8-planner/pull/2710/commits/b17540c1fb013eb90084ac85654b06b686fc724f. Thanks @SMahil